src/JustSaying/Messaging/Channels/Receive/IMessageReceiveController.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace JustSaying.Messaging.Channels.Receive;
+
+/// <summary>
+/// Allows stopping and starting the receiving of messages in all instances of the <see cref="MessageReceiveBuffer"/>
+/// </summary>
+public interface IMessageReceiveController
```

Comment:
Doesn't feel like this really controls anything, it's more of a signal that's observed.